test/unit/app/input-config-tests.js

```diff
@@ -1,281 +0,0 @@
-'use strict';
```

Comment:
Is the difference in # of lines here the removal of the tests around the "required" attribute you were telling me about earlier?